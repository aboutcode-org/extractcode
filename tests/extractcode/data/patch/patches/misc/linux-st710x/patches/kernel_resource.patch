diff -u /home/jonsv322/motorola/DEV_7105_erik_rulez_2/base/3pp/linux-st710x/linux-2.6.23/kernel/resource.c\~ /home/jonsv322/motorola/DEV_7105_erik_rulez_2/base/3pp/linux-st710x/linux-2.6.23/kernel/resource.c
--- linux-2.6.23/kernel/resource.c~	2009-04-07 08:01:25.000000000 +0200
+++ linux-2.6.23/kernel/resource.c	2009-04-27 15:40:19.000000000 +0200
@@ -589,10 +589,6 @@
 	}
 
 	write_unlock(&resource_lock);
-
-	printk(KERN_WARNING "Trying to free nonexistent resource "
-		"<%016llx-%016llx>\n", (unsigned long long)start,
-		(unsigned long long)end);
 }
 EXPORT_SYMBOL(__release_region);
 
