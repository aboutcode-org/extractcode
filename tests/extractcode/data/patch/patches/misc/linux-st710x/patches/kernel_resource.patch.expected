[
  [
    "linux-2.6.23/kernel/resource.c~", 
    "linux-2.6.23/kernel/resource.c", 
    [
      "diff -u /home/jonsv322/motorola/DEV_7105_erik_rulez_2/base/3pp/linux-st710x/linux-2.6.23/kernel/resource.c\\~ /home/jonsv322/motorola/DEV_7105_erik_rulez_2/base/3pp/linux-st710x/linux-2.6.23/kernel/resource.c", 
      "--- linux-2.6.23/kernel/resource.c~", 
      "+++ linux-2.6.23/kernel/resource.c", 
      "@@ -589,10 +589,6 @@", 
      "}", 
      "", 
      "write_unlock(&resource_lock);", 
      "-", 
      "-\tprintk(KERN_WARNING \"Trying to free nonexistent resource \"", 
      "-\t\t\"<%016llx-%016llx>\\n\", (unsigned long long)start,", 
      "-\t\t(unsigned long long)end);", 
      "}", 
      "EXPORT_SYMBOL(__release_region);", 
      ""
    ]
  ]
]